sigh/lib/sigh/resign.rb

```diff
@@ -27,7 +27,10 @@ def resign(ipa, signing_identity, provisioning_profiles, entitlements, version,
       # validate that we have valid values for all these params, we don't need to check signing_identity because `find_signing_identity` will only ever return a valid value
       validate_params(resign_path, ipa, provisioning_profiles)
       entitlements = "-e #{entitlements.shellescape}" if entitlements
-      provisioning_options = provisioning_profiles.map { |fst, snd| "-p #{[fst, snd].compact.map(&:shellescape).join('=')}" }.join(' ')
+      provisioning_options = provisioning_profiles.map do |fst, snd|
```

Comment:
Yeah I'm not sure it ever does something, looking through the code the `provisioning_profiles` var is:

- Is a single string at first [(here)](https://github.com/hjanuschka/fastlane/blob/07aa9e4346d765d30776181e305dff384c73ef04/sigh/lib/sigh/resign.rb#L75)
- Gets passed to the main method [here](https://github.com/hjanuschka/fastlane/blob/07aa9e4346d765d30776181e305dff384c73ef04/sigh/lib/sigh/resign.rb#L8)
- Is then wrapped in an array [here](https://github.com/hjanuschka/fastlane/blob/07aa9e4346d765d30776181e305dff384c73ef04/sigh/lib/sigh/resign.rb#L24)
- As a result, `snd` [here](https://github.com/hjanuschka/fastlane/blob/07aa9e4346d765d30776181e305dff384c73ef04/sigh/lib/sigh/resign.rb#L30) should always be nil, since at this point `provisioning_profiles` is an array of single values

What do you think? Does that sound correct?